src/constants/event-categories.js

```diff
@@ -1,41 +1,58 @@
+// @flow
 // Ideally the keys here should be consts, but currently these are setup in
 // contentful only as the label, hence we need to use this for the key too.
 // If we move to realise categories as its own content type then this will
 // go away and we will fetch this data.
+import * as colors from "./colors";
+import type { EventCategoryMap } from "../data/event";
 
-const categories = {
-  "Cabaret and Variety": {
-    label: "Cabaret and Variety"
-  },
-  Community: {
-    label: "Community"
-  },
-  "Talks and Debates": {
-    label: "Talks and Debates"
-  },
-  "Film and Screenings": {
-    label: "Film and Screenings"
-  },
-  "Plays and Theatre": {
-    label: "Plays and Theatre"
-  },
-  "Social and Networking": {
-    label: "Social and Networking"
-  },
-  Nightlife: {
-    label: "Nightlife"
-  },
-  "Exhibition and Tours": {
-    label: "Exhibition and Tours"
-  },
-  "Sports and Activities": {
-    label: "Sports and Activities"
-  },
-  Health: {
-    label: "Health"
-  },
-  Music: {
-    label: "Music"
+const categories: EventCategoryMap = {
+  "en-GB": {
+    "Cabaret and Variety": {
+      label: "Cabaret and Variety",
+      color: colors.coralColor
+    },
+    Community: {
+      label: "Community",
+      color: colors.brightLightBlueColor
+    },
+    "Talks and Debates": {
+      label: "Talks and Debates",
+      color: colors.darkSkyBlueColor
+    },
+    "Film and Screenings": {
+      label: "Film and Screenings",
+      color: colors.lightTealColor
+    },
+    "Plays and Theatre": {
+      label: "Plays and Theatre",
+      color: colors.warmPinkColor
+    },
+    "Social and Networking": {
+      label: "Social and Networking",
+      color: colors.turquoiseBlueColor
+    },
+    Nightlife: {
+      label: "Nightlife",
+      color: colors.yellowColor,
+      contrast: true
```

Comment:
This flag allows us to controller the colors for accessibility.
For example, the `Nightlife` category:

<img width="394" alt="screen shot 2018-04-06 at 17 23 49" src="https://user-images.githubusercontent.com/822817/38432425-4d0cc990-39bf-11e8-9d12-6b65567a709d.png">